dora/tests/src/test/java/alluxio/client/rest/TestCase.java

```diff
@@ -169,6 +176,7 @@ public HttpURLConnection execute() throws Exception {
     }
 
     connection.connect();
+    connection.getResponseCode();
```

Comment:
why `getResponseCode` here?

Reply:
I use `getResponseCode` to trigger sending the request

Reply:
A confusing explanation. please check the code of `getResponseCode`, it's not for trigger sending the request.

Reply:
I can't found better alternative than `getResponseCode`